zipcodes/france/notebook.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
Ok, just to leave a comment: Let's remove this notebook since it's been merged with the other (if you agree)